teletext.go

```diff
@@ -860,6 +861,7 @@ func parseTeletextRow(i *Item, d decoder, fs func() styler, row []byte) {
 	var l = Line{}
 	var li = LineItem{InlineStyle: &StyleAttributes{}}
 	var started bool
+
```

Comment:
I'm being picky here, but can you not add this line feed? It messes up the git diff

Reply:
Done